web_graph_radar/README.md

```diff
@@ -0,0 +1,4 @@
+Graph Radar Chart.
+=================
+
+Add new graph view mode: Radar
```

Comment:
Use this template for readme
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst